tests/resources/example_docker_project/Dockerfile

```diff
@@ -1,5 +1,9 @@
-FROM continuumio/miniconda3:4.6.14
+FROM condaforge/miniforge3
```

Comment:
This PR speeds up the python-large check because 

- continuumio/miniconda3: based on Debain, we need to build wheels when installing packages.
- ondaforge/miniforge3: based on Ubuntu, we can use pre-built wheels.

Reply:
awesome find, Haru!

Reply:
awesome find + 1